asset/css/default-vars.less

```diff
@@ -0,0 +1,142 @@
+/*
+  RECOMMENDATION:
+      Please do not use the base color variables directly,
+      define a new variable instead that assigns the value of this base variable.
+        Examples:
+         - @base-color: red;
+         - @my-new-var: @base-color;
+         - @my-second-new-var: @base-color;
+
+     Do not use the same variable for different use cases, but define a new variable for each use case.
+
+  NOTICE::
+    Color vars identification:
+       - Vars with `-bg` suffix are background-color vars. Please use only for setting bg-color.
+       - Vars with `-color` suffix color vars. Please use only for setting color.
+*/
+
+@default-bg: #282E39;
+
+@base-gray: #c4c4c4;
+@base-gray-light: #5c5c5c;
+@base-gray-lighter: #4b4b4b;
+@base-disabled: #9a9a9a;
+
+@base-primary-color: #00C3ED;
+@base-primary-bg: #00C3ED;
+
+@default-text-color: #fff;
+@default-text-color-light: fade(@default-text-color, 75%);
+@default-text-color-inverted: @default-bg;
+
+@state-ok: #44bb77;
+@state-up: @state-ok;
+@state-warning: #ffaa44;
+@state-critical: #ff5566;
+@state-down: @state-critical;
+@state-pending: #77aaff;
+@state-unknown: #aa44ff;
+
+@primary-button-color: @default-text-color-inverted;
+@primary-button-bg: @base-primary-bg;
+@primary-button-hover-bg: #0081a6;
+
+@search-term-bg: @base-gray;
+@search-term-color: @default-text-color-inverted;
+@search-term-selected-bg: @base-disabled;
+@search-term-invalid-bg: @state-critical;
+@search-term-invalid-color: @default-text-color-inverted;
+@search-term-disabled-bg: @base-gray-light;
+@search-term-selected-color: @base-gray-light;
+@search-term-highlighted-bg: @base-primary-bg;
+@search-term-highlighted-color: @base-primary-color;
```

Comment:
`@base-primary-bg` and `@base-primary-color` are incompatible. Use `@default-text-color-inverted` instead.